presto-docs/README.md

```diff
@@ -10,6 +11,32 @@ The presto-docs module contains the reference documentation for Presto.
 - [Versioning](#versioning)
 - [Known Issues](#known-issues)
 
+## Writing and contributing
+
+We welcome any contributions to the documentation. Contributions need to [follow
+the same process as code contributions](https://prestosql.io/development/) and
```

Comment:
`Contribution Process` page is more specific?

```suggestion
the same process as code contributions](https://prestosql.io/development/process.html) and
```

Reply:
I was contemplating that but I kinda think thats too specific ...  maybe link to both? Wdyt @electrum ?

Reply:
Linking to the high level page seems better, since it looks cleaner and gives an overall view of the development guidelines. I agree the process page is most relevant, but I think it's better for the reader to have some context before jumping into the list of steps.